screens/CustomerPhoneNumberScreen.js

```diff
@@ -46,30 +39,24 @@ export default class CustomerPhoneNumberScreen extends React.Component {
           maxRecords: 1,
           filterByFormula: `{Phone Number} = '${phoneNumber}'`
         })
-        .eachPage(
-          function page(records, fetchNextPage) {
-            if (records.length == 0) {
-              reject("Incorrect customer phone number. Please try again.");
-            } else {
-              records.forEach(function(record) {
-                // console.log(record["fields"]["First Name"]);
-                resolve(record.getId());
-              });
-            }
-            fetchNextPage();
-          },
-          function done(err) {
-            if (err) {
-              reject(err);
-            }
+        .eachPage(function page(records, fetchNextPage) {
```

Comment:
I believe you can do `.all()` for this; if we only expect one record, then we can just use `.firstPage` instead of `.eachPage`!

Reply:
Using `.firstPage`!